### PR TITLE
HSC-176: Documented custom changes added to official add-on.

### PR DESCRIPTION
**Pull request by Ruhanga**
Ticket: https://mekomsolutions.atlassian.net/browse/HSC-176

### CODE REVIEW

**Comment by Ruhanga** [comment]
@rbuisson, would you please review this PR? Thanks.

We will rebase and merge (and not squash-merge) if all is good.

File: README.md [hunk 1]
```diff
@@ -1 +1,11 @@
...
\ No newline at end of file
+# Odoo Sale Payment Status Add-on (sotto_sale_payment_status)
+
+This add-on aims at providing payment status information on Odoo sale orders. For more information please visit official add-on website/Odoo platform [here](https://apps.odoo.com/apps/modules/14.0/sotto_sale_payment_status/#).
+
+This documentation outlines the modifications made to the "sotto_sale_payment_status" addon in the Odoo platform.
+
+Release notes:
+
+## Version 14.0.1.0.0
```
Comment (rbuisson) [inline README.md]: Could we change to `14.0-1.0.0`?

File: README.md [hunk 2]
```diff
@@ -1 +1,11 @@
...
+* Change Description: Modification: Changed the value of the auto_install attribute to True in the `__manifest__.py`file.
+Purpose of Change: Enhancement: By setting auto_install to True, the addon will now automatically install along with its dependencies when the module is installed thus preventing manual installations through the UI.
```
Comment (rbuisson) [inline README.md]: > By setting `auto_install` to `True` ...

File: README.md [hunk 3]
```diff
@@ -1 +1,11 @@
...
+# Odoo Sale Payment Status Add-on (sotto_sale_payment_status)
+
+This add-on aims at providing payment status information on Odoo sale orders. For more information please visit official add-on website/Odoo platform [here](https://apps.odoo.com/apps/modules/14.0/sotto_sale_payment_status/#).
+
+This documentation outlines the modifications made to the "sotto_sale_payment_status" addon in the Odoo platform.
+
+Release notes:
+
+## Version 14.0.1.0.0
+* Change Description: Modification: Changed the value of the auto_install attribute to True in the `__manifest__.py`file.
```
Comment (rbuisson) [inline README.md]: > Changed the value of the `auto_install` attribute to `True`...

**Review by rbuisson — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (rbuisson) [reply README.md]: @Ruhanga  even though this comment appears in GH as outdated, it seems the feedback was not implemented.

File: README.md (hunk 3 above)
Comment (rbuisson) [reply README.md]: Same here.

Comment (Ruhanga) [reply README.md]: Should be good now, I've just realised you were referring to the code formatting.

File: README.md (hunk 2 above)
Comment (Ruhanga) [reply README.md]: Should be good now.

**Comment by rbuisson** [comment]
Thanks 👍🏻 .

**Review by rbuisson — APPROVED** [review]

**Pull request merged**